src/curve/stable/StablePair.sol

```diff
@@ -302,6 +294,9 @@ contract StablePair is ReservoirPair {
         int112 currLogLiq = ConstantProductOracleMath.calcLogLiq(aReserve0, aReserve1);
         prevClampedPrice = currClampedPrice;
 
+        // overflow is desired here as the consumer of the oracle will be reading the difference in those
+        // accumulated log values
+        // when the index overflows it will overwrite the oldest observation and then forms a loop
```

Comment:
Should mention the case where we have no swaps for N years and the timestamp overflows twice

Reply:
okay added the comment, but added it in ReservoirPair L147 as I think it's a more appropriate venue for this comment